.gitignore

```diff
@@ -0,0 +1,4 @@
+build/
```

Comment:
The builddir can be named anything, so it isn't feasible to predict it in .gitignore and check that into git -- however, Meson does know at the time it produces the build directory, and it also knows that those files should NEVER be checked into git.

So Meson does the helpful thing and creates a file `builddir/.gitignore` (when generating builddir/ as a build directory) which ignores `*` and means you never have to manually add build directories to your gitignore files. :)